src/Serilog.AspNetCore/AspNetCore/RequestLoggingOptions.cs

```diff
@@ -34,6 +35,15 @@ public class RequestLoggingOptions
                 : ctx.Response.StatusCode > 499
                     ? LogEventLevel.Error
                     : LogEventLevel.Information;
+        
+        IEnumerable<LogEventProperty> DefaultGetMessageTemplateProperties(HttpContext httpContext, string requestPath, double elapsedMs, int statusCode) =>
+            new[]
+            {
+                new LogEventProperty("RequestMethod", new ScalarValue(httpContext.Request.Method)),
+                new LogEventProperty("RequestPath", new ScalarValue(requestPath)),
+                new LogEventProperty("StatusCode", new ScalarValue(statusCode)),
+                new LogEventProperty("Elapsed", new ScalarValue(elapsedMs))
```

Comment:
for search requests we already have another name of the variable:
"Search request {OfferSearchId} finished processing in {RequestStatus} after {ElapsedMilliseconds:N0} ms"
should we change it there to Elapsed as well?